README.md

```diff
@@ -123,19 +123,19 @@ Note: If you are running tests on a Mac, run:
 To run tests locally it is as simple as calling <code>py.test</code> with
 several flags. To run testcases that do not modify or delete data:
 
-    py.test --driver=firefox --baseurl=http://mozillians.allizom.org --variables=/full/path/to/variables.json .
+    py.test --driver=Firefox --base-url=http://mozillians.allizom.org --variables=/full/path/to/variables.json .
 
 To run testcases that are known to change or delete account data use the
 <code>--destructive</code> flag:
```

Comment:
We can remove this note about destructive as this was removed in pytest-selenium, now any destructive tests are automatically skipped on sensitive environments like production.